### PR TITLE
i18n(fr) Create `styling.mdx`

### PR DESCRIPTION
**Pull request by McFlyPartages**
#### Description (required)

Create french translation for styling.mdx file

### CODE REVIEW

**Comment by McFlyPartages** [comment]
there are error with check link but i dont know that.

a idea ?

```
19:22:55 [build] Building static entrypoints...
Error: 3 [ERROR] [vite] [@mdx-js/rollup] Expected a closing tag for `<link>` (293:14-293:20) before the end of `strong`
file: /home/runner/work/docs/docs/src/content/docs/fr/guides/styling.mdx:293:3
Expected a closing tag for `<link>` (293:14-293:20) before the end of `strong`
  Stack trace:
    at Object.transform (file:///home/runner/work/docs/docs/node_modules/.pnpm/@astrojs+mdx@2.0.0_astro@4.0.8/node_modules/@astrojs/mdx/dist/index.js:99:27)
    at async ModuleLoader.addModuleSource (file:///home/runner/work/docs/docs/node_modules/.pnpm/rollup@4.6.1/node_modules/rollup/dist/es/shared/node-entry.js:177[24](https://github.com/withastro/docs/actions/runs/7514403403/job/20457235467?pr=6378#step:4:25):36)
 ELIFECYCLE  Command failed with exit code 1.
 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1
```

File: src/content/docs/fr/guides/styling.mdx [hunk 1]
```diff
@@ -0,0 +1,625 @@
...
+<div>
+  <h1>
+    Cette en-tête sera rouge !
+  </h1>
+</div>
+```
+
+Les règles CSS Astro sont évaluées dans cet ordre d'apparition :
+
+- **balises "<link>" dans l'en-tête** (priorité la plus faible)
```
[the remark this answers is not in the document]
Comment (McFlyPartages) [reply src/content/docs/fr/guides/styling.mdx:299]: Lol ligne 293 au.lieu de 299 haha

**Review by thomasbnt — APPROVED** [review]
Oh good! CI Links works now! LGTM!

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Pull request merged**